### PR TITLE
remove duplicated cluster verify logic in ubuntu deployments scripts.

### PR DESCRIPTION
**Pull request by kevin-wangzefeng**
When deploying with Ubuntu scripts, if processes not started correctly on master or minion, the scripts will keep waiting and checking, instead of exit with any error message. With this commit, the scripts will exit with a message like `Warning: Process "docker" status check timeout, please check manually` after 60s waiting and checking.  

---

Update in 2015-9-7
In ubuntu deployment scripts, there are two ways to verfiy the cluster:
    1. `cluster/validate-cluster.sh` called after `kube-up()` in `cluster/kube-up.sh`, is the standard way.
    2. `verify_master()` and `verify_minion()` in `ubuntu/util.sh` which are ssh into node and check if processes are running, are not accurate.

As a result of discussion, this PR is to remove the duplicated cluster verify logic(in `ubuntu/util.sh`) and keep the standard way(calling `cluster/validate-cluster.sh`).

### CODE REVIEW

**Comment by kevin-wangzefeng** [comment]
@dchen1107 @WIZARD-CXY - Hi, could you please review this PR? Thanks.

**Comment by dchen1107** [comment]
I never bring up a ubuntu cluster before, so not sure 60s waiting is enough for a node (master and minions) to be ready or not? Also I noticed that both verify_master and verify_minion are ssh into node and check if processes are running, which is not accurate. Why not use cluster/validate-cluster.sh, the standard way to validate the cluster?

**Comment by WIZARD-CXY** [comment]
@dchen1107 Oh, I wrote the `verify_master` and `verify_minion` long long time ago , maybe in the first version and leave it there.

File: cluster/ubuntu/util.sh [hunk 1]
```diff
@@ -182,6 +191,11 @@ function verify-minion(){
       ssh $SSH_OPTS "$1" "pgrep -f $daemon" >/dev/null 2>&1 || {
         printf "."
         validated="1"
+        ((try_count=try_count+1))
+        if [[ ${try_count} -gt ${max_try_count} ]]; then
+          printf "\nWarning: Process \"${daemon}\" status check timeout, please check manually.\n"
```
Comment (WIZARD-CXY) [inline cluster/ubuntu/util.sh]: Will it change to `\nWarning: Process \"${daemon}\" failed to run on ${MACHINE_IP}, please check.\n`

**Comment by WIZARD-CXY** [comment]
@kevin-wangzefeng good update. Thanks
Recently I may not have time testing your change.Have you tested it with OK?

**Comment by kevin-wangzefeng** [comment]
@WIZARD-CXY Yes, I have tested it. And thanks for the review, I'll update the warning message and test it again.

**Comment by WIZARD-CXY** [comment]
OK

**Comment by dchen1107** [comment]
@kevin-wangzefeng Can you update pr to use cluster/validate-cluster.sh, the stardard way to validate cluster, and terminate after timeout? I think this is a good time to consolidate this instead of adding more logic to different approach, which make the maintainence harder in a long run. Thanks!

**Comment by kevin-wangzefeng** [comment]
@dchen1107 Currently in `cluster/kube-up.sh`, `cluster/validate-cluster.sh` is called after `kube-up()`  returned. It would be duplicated to validate cluster here, so I'm thinking if I can just remove the logic here and keep the standard way (calling `cluster/validate-cluster.sh` after `kube-up()` returned).
What do you think?

**Comment by dchen1107** [comment]
SGTM, that is pretty much what I suggested. Thanks!

**Comment by kevin-wangzefeng** [comment]
@dchen1107, I've updated the PR (squash outdated commits, update the title and description), please review. Thanks!

**Comment by kevin-wangzefeng** [comment]
cc @k8s-oncall

**Comment by dchen1107** [comment]
LGTM

**Comment by kevin-wangzefeng** [comment]
seems to be jenkins timeout

**Comment by kevin-wangzefeng** [comment]
Jenkins timeout again...

**Comment by kevin-wangzefeng** [comment]
@dchen1107 It seems the jenkins cannot be triggered by closing/reopening the PR, could you please help to reply "ok to test" to the robot? Thanks!

**Comment by dchen1107** [comment]
ok to test

**Comment by kevin-wangzefeng** [comment]
shippable failed at persistent_volumes_test.go:109, seems to be a flake.

**Comment by kevin-wangzefeng** [comment]
rebased

**Pull request merged**